IsTheMartin/mrtnmrls
# Images optimised and size reduced

## Summary
Some images were optimised by using Squoosh, they are now lighter.

## Evidence

|Before|After|
|---|---|
|<img width="163" alt="image" src="https://github.com/IsTheMartin/mrtnmrls/assets/38388605/a3f485c1-2195-4f20-acd9-83ef7e5f73a9">|<img width="166" alt="image" src="https://github.com/IsTheMartin/mrtnmrls/assets/38388605/d9d6237f-d23d-4739-a179-c855b40c9109">|

## Concern Information

To test please follow the next steps:

1. Navigate to http://mrtnmrls.com/es/nature-impressive
2. Open inspector, network tab
3. Observe size in resources

## Issue ticket

- https://github.com/IsTheMartin/mrtnmrls/issues/47

## Check list

- [ x ] I did add reviewers to this PR
